Source/Core/Core/BootManager.cpp

```diff
@@ -414,9 +414,7 @@ void Stop()
 
 void RestoreConfig()
 {
-  SConfig::GetInstance().LoadSettingsFromSysconf();
```

Comment:
You removed the call to LoadSettingsFromSysconf() completely -- is this intended?